api-metastore/src/main/java/org/zalando/nakadi/controller/SchemaController.java

```diff
@@ -161,4 +165,21 @@ public ResponseEntity<?> getSchemaVersion(@PathVariable("name") final String nam
         final EventTypeSchema result = schemaService.getSchemaVersion(name, version);
         return ResponseEntity.status(HttpStatus.OK).body(result);
     }
+
+    @GetMapping(
```

Comment:
I would be useful to think about the API first. For that Nakadi repo has a swagger file. Could you make a proposal first for the API change? In regards to `GET` method with body, usually, for body `POST` request is used and it suits here. We have imagined that API endpoint would look like: `/event-types/{name}/schema/latest`